AirCasting/SessionViews/SessionCardView.swift

```diff
@@ -101,7 +101,9 @@ struct SessionCardView: View {
         }
         .onChange(of: selectedStream, perform: { [weak graphStatsDataSource, weak mapStatsDataSource, weak chartViewModel] newStream in
             graphStatsDataSource?.stream = newStream
+            graphStatsDataSource?.dataSource.stream = newStream
```

Comment:
Could this be done on set of stream in converter?

Reply:
not really. The `stream` is a property of both the `ConveringStatisticsDataSourceDecorator` and `Graph/MapStatsDataSource` concrete classes, but not the `MeasurementsStatisticsDataSource` itself. This is also why I exposed the original data source: `let dataSource: D` - in case we need access to the specific stuff that concrete datasources expose.